kasa/tests/fakeprotocol_smart.py

```diff
@@ -234,7 +234,7 @@ def _send_request(self, request_dict: dict):
                 pytest.fixtures_missing_methods[self.fixture_name] = set()
             pytest.fixtures_missing_methods[self.fixture_name].add(method)
             return retval
-        elif method == "set_qs_info":
+        elif method in ["set_qs_info", "fw_download"]:
```

Comment:
Maybe fixtures_missing_methods could also be used for these? Or maybe we want to have separate list for methods returning no reply?

Reply:
This looks ok if `fw_download` just returns an error code.  Fixture missing methods is generally for `get_` and ideally we'll get rid of them eventually